NYPLCardCreator/Flow/UsernameAndPasswordViewController.swift

```diff
@@ -176,10 +197,12 @@ final class UsernameAndPINViewController: FormTableViewController {
         }
 
         guard let response = response as? HTTPURLResponse,
-          response.statusCode == 200,
+          response.statusCode == 200 || response.statusCode == 400,
```

Comment:
can you add a comment to explain why `400` is "ok" here?

Reply:
Done